### ISSUE TITLE
Failing specs

### ISSUE THREAD

**Issue by 50Wliu** [tombell/travis-ci-status#87]
All the specs have been failing since c0fa7e4d96c69319a109be39783e9e7d26d98bb8, including the commit where you released version 1.3.0.  That doesn't make me feel too confident about whether or not travis-ci-status is working as expected...

**Comment by ghost1227** [comment]
Yep! The package works, I'm just still learning how to write atom packages. If you're concerned, please feel free to revert to the previous version until I fix it (or submit a PR)
